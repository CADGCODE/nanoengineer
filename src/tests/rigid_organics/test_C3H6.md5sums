stdout A14FE025D15C6AAD6534E60EACD7E89F
stderr 160E6504290AEB20212DA51B283BAFE4
test_C3H6.trc 883A2342DCB4EC495081B6F3C9060A87
test_C3H6.xyz 6B902F3F0F7CC5D22A1D21CAB7907025
