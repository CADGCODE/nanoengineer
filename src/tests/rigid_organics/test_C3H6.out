======= test_C3H6.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
cutover to fine tolerance at 0
== structure comparison ==
